model/dce-poll.cc

```diff
@@ -36,7 +36,7 @@ int dce_poll (struct pollfd *fds, nfds_t nfds, int timeout)
     }
   else if (timeout > 0)
     {
-      endtime = Now () + MilliSeconds (timeout);
+      endtime = Now () + MicroSeconds (timeout);
```

Comment:
the manpage does say it should be Milliseconds so I believe this change is incorrect

Reply:
Good catch, I will ask Trish to weigh in about this.  It would be nice to have some test code..